### ISSUE TITLE
cron Connection refused

### ISSUE THREAD

**Issue by hmaesta** [danschultzer/monit-graph#5]
Everything is fine when `monit -t` and I can access the default interface (`domain.com:2812`), but I can't use MonitGraph (`domain.com/mg`).

Checking my spool, I noticed that MonitGraph is sending: 

> [MonitGraph] /var/www/domain.com/htdocs/mg/monit-graph.class.php line 154: cURL Error (7): Failed to connect to localhost port 2812: Connection refused

And accessing `domin.com/mg` I receive 403.

Nginx log just say "directory forbidden". All chmod and users are fine, what can be wrong?

**Comment by danschultzer** [comment]
Sounds like Monit is not running with the web interface?

https://github.com/danschultzer/monit-graph#edit-configuration-file-for-monit

**Comment by hmaesta** [comment]
Monit is running with web interface (I can access `domain.com:2812`).

Everything is like the configuration you give, except the `pemfile` part (mine is pointing to `/etc/ssl/monit.pem`) because I had to generate another one, [following here](http://serverfault.com/questions/567426/monit-with-ssl-https).

Is there another log I can check?

**Comment by danschultzer** [comment]
Sounds like there's two issues at hand:

**1) Monit Graph load issue**

What's the following options set to in `config.php`?
- `url`
- `url_ssl` (should be true)
- `verify_ssl` (should be false)

**2) Nginx**

What's your nginx config? You should have something along the lines of:

```
 location ~ [^/]\.php(/|$) {
     fastcgi_split_path_info ^(.+?\.php)(/.*)$;
     if (!-f $document_root$fastcgi_script_name) {
         return 404;
     }

     fastcgi_pass 127.0.0.1:9000;
     fastcgi_index index.php;
     include fastcgi_params;
 }
```

**Comment by hmaesta** [comment]
My configuration files appears to be fine.
I just disabled SSL for Monit (`:2812`), tried `curl login:pass domain.com:2812` on my local machine and the response was correct. _(I changed 'localhost' by 'domain.com' in config file for this test)_

On Monit Graph's  `config.php` file, the only change was on `url_ssl`: set it to false (since I disabled SSL for Monit).

My nginx config is as simples as it could be, courtesy of [easyengine.io](easyengine.io). 

I really don't know why I am receiving Connection refused. :(
